### PR TITLE
Agroeasy

### PR DESCRIPTION
**Pull request by joshuaobinna**
This is the code we've written

### CODE REVIEW

**Comment by Way2nnadi** [comment]
Delete `Argo_easy/README.md` and `Argo_easy/package-lock.json`

File: Agro_easy/public/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Stock Inventory</title>
+  </head>
+  <body>
+    <script src="accessories/jquery.min.js"></script>
```
Comment (Way2nnadi) [inline Agro_easy/public/index.html]: do you actually have a `jquery.min.js` in this repo. If not remove this line

File: Agro_easy/public/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (Way2nnadi) [inline Agro_easy/public/index.html]: remove line `28-37`

File: Agro_easy/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (Way2nnadi) [inline Agro_easy/public/manifest.json]: What purpose does this `manifest.json` serve? If you are not using it remove it from the repo

File: Agro_easy/src/App.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import About from './components/about';
+import Home from './components/home';
+
+
```
Comment (Way2nnadi) [inline Agro_easy/src/App.js]: Add a comment to describe this component. Use this as a reference (https://react-styleguidist.js.org/docs/documenting.html)

File: Agro_easy/src/components/footer.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Col, Row, Container } from 'reactstrap';
+import './home.css';
+import ProductsAvailable from './productsavailable';
+
```
Comment (Way2nnadi) [inline Agro_easy/src/components/footer.js]: Component needs a comment decription

File: Agro_easy/src/components/anotherlist.js [hunk 6]
```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { Col, Row, Container, Jumbotron } from 'reactstrap';
+import { Link } from 'react-router-dom';
+import './home.css';
+
+class AnotherList extends React.Component {
```
Comment (Way2nnadi) [inline Agro_easy/src/components/anotherlist.js]: Component needs a comment description

File: Agro_easy/src/components/home.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+import { InputGroup, InputGroupAddon, Jumbotron, Col,
+         Button, Input, Row, Container
+} from 'reactstrap';
+import Footer from './footer';
+import AnotherList from './anotherlist';
+import Navibar from './Navibar';
+import './home.css';
+import './bootstrap.min.css';
+
+ class Home extends React.Component {
```
Comment (Way2nnadi) [inline Agro_easy/src/components/home.js]: Component needs a comment description

File: Agro_easy/src/components/productsavailable.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Col, Row } from 'reactstrap';
+import './home.css';
+
+
+export default class ProductsAvailable extends Component {
```
Comment (Way2nnadi) [inline Agro_easy/src/components/productsavailable.js]: Component needs a comment description

File: Agro_easy/src/App.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+class App extends Component {
+    render() {
+        return (
+            <Router>
+                <div className="App">
+                   <Route path='/' exact strict component={Home} />
```
Comment (Way2nnadi) [inline Agro_easy/src/App.js]: what does adding `exact strict` do for the Route component here?

Comment (joshuaobinna) [reply Agro_easy/src/App.js]: it ensure a page renders correclly, like /about/

**Pull request merged**